course/create_course_distribution.md

```diff
@@ -0,0 +1,51 @@
+# Instructions for creating distribution for the OpenDA course
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=4902594)